test/e2e/logforwarding/loki/forward_to_loki_test.go

```diff
@@ -14,11 +14,24 @@ import (
 	"github.com/openshift/cluster-logging-operator/test/runtime"
 )
 
-func TestLogForwardingToLoki(t *testing.T) {
-	c := client.ForTest(t)
-	rcv := loki.NewReceiver(c.NS.Name, "loki-receiver")
+func TestLogForwardingToLokiWithFluentd(t *testing.T) {
+	t.SkipNow()
+	cl := runtime.NewClusterLogging()
+	clf := runtime.NewClusterLogForwarder()
+	testLogForwardingToLoki(t, cl, clf)
+}
+
+func TestLogForwardingToLokiWithVector(t *testing.T) {
 	cl := runtime.NewClusterLogging()
+	cl.Spec.Collection.Logs.Type = loggingv1.LogCollectionTypeVector
+	cl.Spec.Collection.Logs.FluentdSpec = loggingv1.FluentdSpec{}
```

Comment:
why do we need FluentdSpec{} ?

Reply:
just ensuring to not have any values here.